src/odemis/driver/compositer.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 9 May 2025
+
+@author: Éric Piel
+
+Copyright © 2025 Éric Piel, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+
+import logging
+from typing import Dict, Any
+
+from odemis import model
+
+
+HIDDEN_VAS = {"children", "dependencies", "affects"}
+
+class CompositedComponent(model.HwComponent):
+    """
+    Wrapper component to merge multiple components together, seen as one, providing all the
+    VigilantAttributes from all the components.
```

Comment:
Add comment about `metadata`? that the metadata value and update is for the base dependency component?